ddtrace/appsec/_api_security/api_manager.py

```diff
@@ -128,8 +128,12 @@ def _schema_callback(self, env):
 
         try:
             # check both current span and root span for sampling priority
+            # if sampling has not yet run for the span, we default to treating it as sampled
+            if root.context.sampling_priority is None and env.span.context.sampling_priority is None:
+                priorities = (1,)
```

Comment:
?
```suggestion
                priorities = (1,1)
```

Reply:
In order to make it a tuple, you ned to put a comma after the first member, however we never need to priorities to have anything besides just a single 1 in it, since all we do is check if various values are inside of it, having multiple 1s is the same as having one 1.